docs/securing_client_connection.rst

```diff
@@ -225,6 +226,28 @@ The following is an example configuration:
         ssl_ciphers="DHE-RSA-AES128-SHA:DHE-RSA-AES256-SHA"
     )
 
+Checking Hostname
+^^^^^^^^^^^^^^^^^
+
+During the TLS/SSL handshake, the client can verify that the hostname
+or the IP address of the member matches with the information provided
+in the member's certificate's Subject Alternative Name extension or 
```

Comment:
How about "... in the Subject Alternative Name extension of the member's certificate ..."?